actnow/petitions/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+
+from .models import Petition, PetitionSignature
+
+
+class PetitionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Petition
+        fields = "__all__"
+        depth = 1
```

Comment:
I think we should exclude some fields from the owner model.
![Screenshot 2021-06-17 at 04 35 50](https://user-images.githubusercontent.com/13068580/122317049-8a8a6f80-cf25-11eb-9c69-8a26ac66b1b1.png)